reader/src/main/java/org/kaazing/monitoring/reader/api/GatewayCounters.java

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
This is used in the viewer and statsD publisher. (for the moment, the gateway does not expose any gateway counters, but it will, in the future)